Makefile

```diff
@@ -21,37 +21,27 @@
 
 # TODO: Makefile will be deleted once the repo is public. Check issue #411.
 
-main: client
-	go build ./cmd/keytransparency-server
-	go build ./cmd/keytransparency-signer
+main: 
+	go build ./cmd/keytransparency-server ./cmd/keytransparency-signer ./cmd/keytransparency-client
```

Comment:
Hmm.  I think the useful error is what file had the problem, not necessarily which binary. 
The idea is that we'll get a speedup from only compiling the common dependencies only once.